README.md

```diff
@@ -79,7 +77,7 @@ For example:
 
    ```sh
    aws s3 sync "s3://my-bucket/coverage/$SHA" coverage/
-   cc-test-reporter sum-coverage --output - coverage/codeclimate.*.json | \
+   cc-test-reporter sum-coverage --output - --parts <number-partials-results> coverage/codeclimate.*.json | \
```

Comment:
Can we use some example `$VARIABLE` and explain it after, like we do above for `$N` and `$SHA`?